frontend/package.json

```diff
@@ -12,8 +12,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
-    "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
+    "react-scripts": "5.0.1"
```

Comment:
Whenever you change dependencies, make sure to commit the changes in the lockfile as well.

Reply:
done.